src/layouts/IndexLayout.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import TEMP_FEATURED_DATA from "./../../projects.json";
+import ProjectPreview from "./../components/project/ProjectPreview";
+import Button from "../components/general/Button.js";
+
+function renderProjects() {
+  // TEMPORARY DATA
+  return TEMP_FEATURED_DATA.projects.map(project => {
+    return <ProjectPreview data={project} key={project.name}></ProjectPreview>;
```

Comment:
Projects will eventually have unique IDs. Could we do one of the following?

1. Add fake `id`s to the sample project data, and then use that for the `key` here
1. Add a TODO, saying that we intend to use the `id` for the `key` instead